### DISCUSSION TITLE
[Feature] Show All Albums

### DISCUSSION THREAD

**Discussion by koinupapi** [immich-app/immich#3963] [Feature Request]
### The feature

In Google Photos: there is a "Library" tab, and within that, there is a "Photos on this Device" section where it shows ALL of the albums on my phone, whether or not their being backed up to Google. In this way, it can act as a gallery for my local photos, as well as a gallery for my backed up photos.

In Immich: there is a "Library" tab, and within that, there is an "Albums on Device" section. Currently, it seems to ONLY show albums that have been selected to be backed up to my server. So, it seems that it can ONLY be a gallery viewer for backed up photos, and not for local photos.

It would be nice if we could view all albums, even if not selected to backup, and use Immich as a general Gallery app, just like Google Photos.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by EnochPrime** [comment]
Duplicate of #2117
